src/js/components/Diagram/index.d.ts

```diff
@@ -4,10 +4,24 @@ import { ColorType } from '../../utils';
 
 export type DiagramConnectionAnchor = 'center' | 'vertical' | 'horizontal';
 export type DiagramConnectionType = 'direct' | 'curved' | 'rectilinear';
+export type DiagramAnimationType =
+  | boolean
+  | 'pulse'
+  | 'draw'
+  | {
+      type?: 'pulse' | 'draw';
+      // looking into it, but delay seems to only work when delay <= 1000
```

Comment:
What are the findings? The comment shouldn't be placed here.

Reply:
Just tried out a delay again on the story I was working on and it works as long as you don't run into common things like giving a delay longer than the duration or timer. Pushed up a commit getting rid of the comment on the delay as well as the redundant comment left on the `size` attribute since it is included on the `doc` already.